jobs/acceptance-tests-brain/templates/pre-start.erb

```diff
@@ -4,7 +4,9 @@ set -o errexit
 set -o nounset
 
 PATH=$PATH:/var/vcap/packages/cli/bin
+API="<%= properties.smoke_tests.api %>"
+PASSWORD="<%= properties.acceptance_tests_brain.password %>"
 
-cf api --skip-ssl-validation "https://api.$DOMAIN"
-cf auth admin "${CLUSTER_ADMIN_PASSWORD}"
+cf api --skip-ssl-validation "${API}"
+cf auth admin "${PASSWORD}"
```

Comment:
It looks like there's a [`acceptance_tests_brain.user`](https://github.com/SUSE/brain-tests-release/blob/fe15bb7fa6e9f681644dbb02517296ae9af0121e/jobs/acceptance-tests-brain/spec#L31) that we might want to use.

Reply:
True. Will do.

Reply:
Addressed with commit 39e2b8112e304c5d8b3310bb13cb94fce61c83e7.